net.jammr.jammr.json

```diff
@@ -0,0 +1,135 @@
+{
+    "id": "net.jammr.jammr",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "jammr",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all"
+    ],
+    "cleanup": [
+        "/include",
+        "/share/man",
+        "*.a",
+        "*.la",
+        "/lib/pkgconfig"
+    ],
+    "modules": [
+        {
+            "name": "icons",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 jammr.png /app/share/icons/hicolor/48x48/apps/net.jammr.jammr.png",
+                "install -Dm644 jammr.svg /app/share/icons/hicolor/scalable/apps/net.jammr.jammr.svg"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "jammr.png"
+                },
+                {
+                    "type": "file",
+                    "path": "jammr.svg"
+                }
+            ]
+        },
+        {
+            "name": "appdata",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 net.jammr.jammr.appdata.xml /app/share/metainfo/net.jammr.jammr.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "net.jammr.jammr.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html